qa/test-harness/test-harness.py

```diff
@@ -13,16 +13,13 @@
 
 KAFKA_CONSUMER_TIMEOUT = 10000
 KAFKA_PORT = '9092'
-ODE_PORT = '8080'
-ODE_REST_ENDPOINT = '/upload/bsmlog'
 
 DOCKER_HOST_IP=os.getenv('DOCKER_HOST_IP')
```

Comment:
DOCKER_HOST_IP is also used to set the Kafka subscription. Should that be switched over to `ode_upload_url` as well?

Reply:
I don't think so. I think to subscribe to Kafka, we still need the local IP of the host where kafka broker is running. For now anyway. In the future, we may need to revisit the use of the Host IP for accessing Kafka if Kafka broker is deployed separately.